airbyte-integrations/connectors/source-instagram/source_instagram/streams.py

```diff
@@ -16,14 +16,16 @@
 from facebook_business.exceptions import FacebookRequestError
 from source_instagram.api import InstagramAPI
 
-from .common import remove_params_from_url
+from .common import fix_nested_timestamp, remove_params_from_url
 
 
 class InstagramStream(Stream, ABC):
     """Base stream class"""
 
     page_size = 100
     primary_key = "id"
+    # Define fields to fix as tuples representing the path to the field
+    fix_timestamp_fields = []
```

Comment:
maybe we could remove this attribute and use schema as a source of truth in order to find `date-time` fields. (take a look at the facebook pages implementation)

Reply:
Fixed